src/components/SidebarState.vue

```diff
@@ -34,72 +80,125 @@ export default Vue.extend({
             // nextTick is needed because app_width is 0 on created()
             let sidebarDefault = this.$state.setting('sidebarDefault');
             if (sidebarDefault && this.$state.ui.app_width > 769) {
-                this.sidebarSection = sidebarDefault;
-                this.sidebarOpen = true;
+                this.channelState.sidebarSection = sidebarDefault;
+                this.channelState.sidebarOpen = true;
+                this.queryState.sidebarOpen = true;
             }
         });
     },
     methods: {
         section() {
+            if (this.sidebarOpen === null) {
+                this.sidebarOpen = this.channelState.sidebarOpen;
+            }
```

Comment:
I can't tell what this if block is doing? If null, then it... calls the getter to get sidebarOpen and then sets itself back to its own value?

Reply:
related to above, null means query has not been opened yet, so get the state from channel